### DISCUSSION TITLE
Prometheus Latency Metrics for Services

### DISCUSSION THREAD

**Discussion by saby1101** [nginx/kubernetes-ingress#2313] [Q&A]
I have deployed the ingress controller to GKE. I have enabled latency metrics using the `-enable-latency-metrics` argument. However, there is no metrics being generated for any of the k8s ingresses that are created. From the docs (https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/prometheus/), it looks like the controller_upstream_server_response_latency_ms_count should be available. I am using open source nginx and NOT nginx plus. Are the latency metrics for each ingress available only in the nginxplus version?

**Comment by brianehlert** [comment]
Since this appears global.  Did you also: https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#cmdoption-enable-prometheus-metrics

**Reply by brianehlert** [reply]
also assuming you are generating traffic.

**Comment by ebuildy** [comment]
This is available only for Nginx Plus customers :-/

**Reply by AssafKatz3** [reply]
So we need to document this fact...
